server/sonar-server/src/main/java/org/sonar/server/plugins/edition/EditionInstaller.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.server.plugins.edition;
```

Comment:
hard to find strong arguments, but I would have used "org.sonar.server.editions.plugins"

Reply:
actually, juste noticed that error message refer to installation of edition rather than to installation of plugin

I'd say one point in favor of using "edition.plugins" rather that "plugins.edition" :)

Reply:
one point in favor of 'plugins.edition' is to have these classes next of the ones in 'plugins'. They share hierarchy, it's kind of an specialization of plugins.